deepchecks/recommender/checks/cold_start_detection.py

```diff
@@ -0,0 +1,129 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Outlier detection functions."""
+import time
+from typing import List, Union
+
+import numpy as np
+from PyNomaly import loop
+import pandas as pd
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core.errors import (DeepchecksProcessError, DeepchecksTimeoutError, DeepchecksValueError,
+                                    NotEnoughSamplesError)
+from deepchecks.tabular import  SingleDatasetCheck
+from deepchecks.recommender import  Context
+
+from deepchecks.utils.dataframes import select_from_dataframe
+from deepchecks.utils.strings import format_number, format_percent
+from deepchecks.utils.typing import Hashable
+
+__all__ = ['ColdStartDetection']
+
+DATASET_TIME_EVALUATION_SIZE = 100
```

Comment:
not used

Reply:
fixed.
https://github.com/deepchecks/deepchecks/pull/2610/commits/6c5ee401c6b425ed50cfb89d66303b1260924102